src/Utilities/universalfallback.jl

```diff
@@ -2,17 +2,18 @@
     UniversalFallback
 
 The `UniversalFallback` can be applied on a [`MathOptInterface.ModelLike`](@ref)
-`model` to create the model `UniversalFallback(model)` supporting *any*
-constaint and attribute. This allows to have a specialized implementation in
-`model` for performance critical constraints and attributes while still
+`model` to create the model `UniversalFallback(model)` supporting *any* objective,
```

Comment:
Technically, the objective is an attribute. The objective attribute needs a special handling by `UniversalFallback` but it's still an attribute.